biapol_utilities/utilities/_list_utilities.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+
+import numpy as np
+
+
+def sort_list_pairs(list1, list2, **kwargs):
+    """
+    Sort numeric list according to values in a second array.
+
+    This function orders the values in `list2` according to the values passed
+    as `list1`. The contents can be every sortable format.
+    Code adapted from [0]
+
+    Parameters
+    ----------
+    list1 : list
+        List of values that serve as a reference
+    list2 : list
+        List of values that are sorted according to list1.
+
+    *order*: str, optional
```

Comment:
You are using the `*` star sometimes in parameters. Is there a reference where it's explained why you do that? Happy to learn something here!